src/exts/git.py

```diff
@@ -0,0 +1,150 @@
+import interactions
+from datetime import datetime
+import re
+import aiohttp
+
+
+class Git(interactions.Extension):
+    """An extension dedicated to linking PRs/issues."""
+
+    def __init__(self, bot):
+        self.bot: interactions.Client = bot
+        self.url = "https://api.github.com/repos/interactions-py/library/issues/"
+        self.headers = {"accept": "application/vnd.github.v3+json"}
+
+    @interactions.extension_listener(name="on_message_create")
+    async def on_message_create(self, message: interactions.Message):
+        tags = [tag.strip("#") for tag in message.content.split() if tag.startswith("#")]
+        if message.author.id == self.bot.me.id or message.author.bot or tags[0] == "" or not tags[0].isnumeric():
+            return
+
+        async with aiohttp.ClientSession() as session:
```

Comment:
Isn't it typically bad practice to create an `ClientSession` every time something's run, especially when this could be happening often? I feel like the session could be created in the `__init__` (and then closed during teardown).

Reply:
I'm going to test that, one sec.

Reply:
It is, but the chances people are referencing a github issue or pr every 3 seconds are astronomically low

Reply:
Can't get it to work lmao.

Reply:
Fair enough. It's pretty simple to get working... but it also isn't a big deal.